bedrock/firefox/templates/firefox/enterprise/index.html

```diff
@@ -72,7 +72,11 @@ <h3>{{ _('Firefox Rapid Release') }}</h3>
     </div>
     <div class="c-links">
       <h3>{{ _('Firefox Extended Support Release (ESR)') }}</h3>
-      <a class="qa-download-esr" href="{{ firefox_url('desktop', 'all', 'esr') }}">{{ _('Download ESR') }}</a>
+      <a class="qa-download-esr" href="{{ firefox_url('desktop', 'all') }}">{{ _('Download ESR') }}</a>
+    </div>
+    <div class="c-links">
+      <h3>{{ _('Firefox MSI Installers') }}</h3>
+      <a class="qa-download-msi" href="{{ firefox_url('desktop', 'all') }}">{{ _('Download MSI') }}</a>
```

Comment:
👏 love that you included `qa-download-msi`! One quick note is that we will need to add a test for this link in `tests/pages/firefox/enterprise.py`:
https://github.com/mozilla/bedrock/blob/af951623c9b468b16ebf9711d7bccece3276c13f/tests/pages/firefox/enterprise.py#L16
We can essentially copy the `_download_esr_locator` line for msi. 😄

Reply:
👍  added